### PR TITLE
Fix test-azure-frs pipeline

### PR DESCRIPTION
**Pull request by scottn12**
## Description

This PR fixes lingering issues from https://github.com/microsoft/FluidFramework/pull/15590. Currently, tests are passing running e2e tests against FRS. One test is failing running against tinylicious, however this seems to be an issue with the code rather than the test itself, so this PR disables it and it will be fixed in a separate PR.

This PR also removes `fluid-framework` as a dependency in scenario runner and e2e packages since it causes issues with test pipelines.

## Misc
[AB#3934](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/3934)

### CODE REVIEW

File: azure/packages/test/end-to-end-tests/src/test/containerCreate.spec.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ describe("Container create scenarios", () => {
 	 *
 	 * Expected behavior: an error should be thrown when trying to get a non-existent container.
 	 */
-	it("cannot load improperly created container (cannot load a non-existent container)", async () => {
+	it.skip("cannot load improperly created container (cannot load a non-existent container)", async () => {
```
Comment (sonalideshpandemsft) [inline azure/packages/test/end-to-end-tests/src/test/containerCreate.spec.ts:129]:
```suggestion
	it.skip("cannot load improperly created container (cannot load a non-existent container)", async () => {
```
can we add some `TODO` note above this skipping tests

**Review by sonalideshpandemsft — APPROVED** [review]
lgtm 🥳 

thanks @scottn12!!

File: azure/packages/test/end-to-end-tests/src/test/AzureClientFactory.ts [hunk 2]
```diff
@@ -33,7 +33,7 @@ export function createAzureClient(
 		name: userName ?? uuid(),
 	};
 	const endPoint = process.env.azure__fluid__relay__service__endpoint as string;
-	if (endPoint === undefined) {
+	if (useAzure && endPoint === undefined) {
```
Comment (alexvy86) [inline azure/packages/test/end-to-end-tests/src/test/AzureClientFactory.ts:36]: Maybe just curiosity, but what does it mean for useAzure to be false here? Which service is used in that case?

**Review by alexvy86 — APPROVED** [review]
LGTM overall. The one question below could maybe result in other changes, depending on the answer, but they probably can be addressed separately if that's the case.

**Pull request merged**